src/components/Articles/ViewArticle.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { fetchSpecificArticle } from '../../actions/articleActions';
+import ReadArticle from './ReadArticle';
+
+export class ViewArticle extends Component {
+  componentDidMount() {
+    const { match: { params: { slug } } } = this.props;
+    const { fetchSpecificArticle } = this.props;
```

Comment:
Line 10 and 11 could be declared at once like:
`const { match: { params: { slug } }, fetchSpecificArticle } = this.props;`